%YAML:1.0
opencv_ml_svm:
   format: 3
   svmType: C_SVC
   kernel:
      type: RBF
      gamma: 1.
   C: 1.
   term_criteria: { epsilon:1.1920928955078125e-07, iterations:1000 }
   var_count: 3
   class_count: 2
   class_labels: !!opencv-matrix
      rows: 2
      cols: 1
      dt: i
      data: [ -1, 1 ]
   sv_total: 33
   support_vectors:
      - [ 1.30926952e-01, 6.14096105e-01, 1.05195594e+00 ]
      - [ -1.71779826e-01, 1.23661327e+00, -1.58753836e+00 ]
      - [ 8.85202229e-01, -1.17646003e+00, -1.89987905e-02 ]
      - [ -8.54257122e-02, 3.92994165e-01, 9.46554601e-01 ]
      - [ -8.32252622e-01, -1.48311658e-02, 5.51252782e-01 ]
      - [ 4.80508879e-02, 1.10555279e+00, -1.47383595e+00 ]
      - [ 3.72274709e-03, 7.44170010e-01, 9.91283655e-01 ]
      - [ 1.10033011e+00, -4.66909200e-01, 1.04932757e-02 ]
      - [ -1.09454679e+00, -5.52540366e-03, -2.05774593e+00 ]
      - [ -1.62748039e-01, 3.53763431e-01, 8.55470002e-01 ]
      - [ -3.72143060e-01, 2.87001419e+00, 1.17620122e+00 ]
      - [ 2.26020589e-01, 7.24273384e-01, 1.04654419e+00 ]
      - [ -2.21953407e-01, -1.60870862e+00, 1.05990458e+00 ]
      - [ 1.61377922e-01, 6.68329656e-01, 1.06157589e+00 ]
      - [ 2.29136180e-02, 7.32137680e-01, 9.23108876e-01 ]
      - [ -2.48540252e-01, 1.69316307e-01, -8.23416889e-01 ]
      - [ -6.13681495e-01, -1.34719849e+00, 8.06577802e-01 ]
      - [ 4.25427333e-02, 3.63614470e-01, 8.01924229e-01 ]
      - [ -1.07043135e+00, -1.40612185e-01, -7.93883204e-01 ]
      - [ -3.61757763e-02, 6.59088135e-01, 9.71380770e-01 ]
      - [ -1.69770643e-01, 3.58261883e-01, 8.73930752e-01 ]
      - [ -1.42816171e-01, 3.78870845e-01, 9.41484511e-01 ]
      - [ 4.25848998e-02, 3.00466627e-01, 9.75259662e-01 ]
      - [ -1.71845987e-01, 3.47825587e-01, 6.64169133e-01 ]
      - [ -5.85209355e-02, -1.50234139e+00, 1.04974174e+00 ]
      - [ -6.92900181e-01, 7.38820910e-01, 5.27113438e-01 ]
      - [ -7.44125187e-01, 7.43663967e-01, 5.25474072e-01 ]
      - [ 6.87288120e-02, 3.56755674e-01, 9.98521268e-01 ]
      - [ -8.21555972e-01, 6.44649327e-01, -1.93738210e+00 ]
      - [ 4.55879979e-03, 2.73940659e+00, 1.16424310e+00 ]
      - [ -1.41297770e+00, -9.58071873e-02, -1.84190285e+00 ]
      - [ 1.78841138e+00, -6.50005519e-01, -3.23822677e-01 ]
      - [ -2.05438066e+00, -3.00599456e-01, -7.51228869e-01 ]
   decision_functions:
      -
         sv_count: 33
         rho: 6.6642829962171257e-01
         alpha: [ 6.7924733906023682e-01, 5.8737574402125869e-01, 1., 1.,
             1., 8.8031034827874621e-01, 6.1118319403881105e-01, 1.,
             5.0510432166996855e-01, 1., 1., 2.5085835212245117e-01, -1.,
             -1.3382283030708109e-01, -4.2738634681069254e-02,
             -1.6786604408045389e-02, -2.6112487906005073e-02,
             -4.2610693899664737e-02, -1., -7.8748082538761777e-01,
             -2.7781980728304995e-01, -2.0925268207611361e-03, -1.,
             -2.3639986887183498e-01, -3.3273115480290799e-01,
             -4.6341161460281627e-01, -1., -3.1082782500016359e-01, -1.,
             -2.5885883428741974e-01, -2.0917334960954373e-01, -1.,
             -3.7321224132349140e-01 ]
         index: [ 23, 24, 3, 27, 22, 4, 12, 17, 16, 20, 21, 9, 19, 18,
             29, 30, 28, 31, 25, 26, 32, 5, 0, 1, 2, 11, 14, 15, 13, 10,
             8, 6, 7 ]
...
---
mean: !!opencv-matrix
   rows: 1
   cols: 3
   dt: f
   data: [ 1.46941757e+02, 1.35695251e+02, 1.34505692e+02 ]
std: !!opencv-matrix
   rows: 1
   cols: 3
   dt: f
   data: [ 5.69389992e+01, 1.16944742e+01, 1.82059479e+01 ]
